zerver/webhooks/raygun/doc.md

```diff
@@ -0,0 +1,30 @@
+Raygun is an error and performance monitoring software for web and mobile apps.
+Follow these steps to integrate your Raygun application with Zulip:
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+{!append-topic.md!}
+
+Copy the URL and do the following steps:
+
+1. Go to your [Raygun dashboard](https://app.raygun.com/dashboard/) and select
+   your application (top left):
+   ![](/static/images/integrations/raygun/001.png)
+
+2. Click on Integrations at the left sidebar:  
```

Comment:
* Replace `Integrations` with `**Integrations**`.
* Make it `on the left sidebar`. :)

Reply:
Okay!